instrumentation/opentelemetry-instrumentation-falcon/src/opentelemetry/instrumentation/falcon/__init__.py

```diff
@@ -195,12 +195,23 @@ def __call__(self, env, start_response):
 
         start_time = _time_ns()
 
-        token = context.attach(extract(env, getter=otel_wsgi.wsgi_getter))
+        token = ctx = span_kind = None
+
```

Comment:
this snippet is becoming pretty common. probably could take it out as a re-usable function.

Reply:
Added re-usable getter function for the three variables. Could not think of better name. Please suggest.